error.go

```diff
@@ -2,38 +2,42 @@
 // and efficient error handling.
 package microerror
 
-import (
-	"fmt"
-
-	"github.com/juju/errgo"
-)
-
 var (
-	// MaskAny is a simple error masker. Masked errors act as tracers within the
-	// source code. Inspecting an masked error shows where the error was passed
-	// through within the code base. This is gold for debugging and bug huntin.
-	MaskAny = errgo.MaskFunc(errgo.Any)
+	Default = NewErrgoHandler(ErrgoHandlerConfig{
+		CallDepth: 1,
```

Comment:
I see. Now I understand. I think this is bad. You change the default of the whole package, for everybody using this package. Imagine somebody does this with a dependency you are relying on and everything ends up in your project's vendor directory. Then you wonder why something is fishy and after some hours or even days it turns out somebody screwed the whole package. I think you should just work against your own instantiation and we should aim to prevent package wide globals. This is also one thing I hate on all the logging packages. This is not deterministic and at the end you rely on everybody's "intelligence" to not screw your own code.